coap-core/src/main/java/com/mbed/coap/server/internal/CoapUdpMessaging.java

```diff
@@ -103,10 +108,43 @@ public final void init(int duplicationListSize,
             long delayedTransactionTimeout,
             DuplicatedCoapMessageCallback duplicatedCoapMessageCallback) {
 
+        if (duplicationListSize > 0) {
+            cache = new DefaultDuplicateDetectorCache<CoapRequestId, CoapPacket>(
```

Comment:
This should be done in the builder and we won't need 3 init() methods then. It is enough to have the last one that gets the cache itself.

Reply:
This is now done.